drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/aha.rb

```diff
@@ -76,10 +78,13 @@ def fetch_score(client)
         end
       end
 
+      # Find the highest AHA score
       highest_aha_score = score_objects.compact.
         filter { |score_object| score_object.generator == AHA_GENERATOR }.
         max_by(&:score)
-      return nil if highest_aha_score.nil? || highest_aha_score.score&.negative?
```

Comment:
This change is so that when the API returns a score of -1, we can still return and store the other fields (generator, dw_client_id, etc) so we have those for reporting.